examples/tutorials/4-InARow.bgl

```diff
@@ -4,10 +4,10 @@
 
 game InARow
 
```

Comment:
these type definitions used `P` in two type definitions, which is no longer valid. It must be done by reference to `P`'s type name,